### PR TITLE
Add Network Manager Prereqs

### PR DESCRIPTION
**Pull request by satmandu**
- Also updated Jansson

Builds properly:
- [x] x86_64
- [x] armv7l
- [x] i686

### CODE REVIEW

File: packages/mobile_broadband_provider_info.rb [hunk 1]
```diff
@@ -0,0 +1,37 @@
+require 'package'
+
+class Mobile_broadband_provider_info < Package
+  description 'Network Management daemon'
+  homepage 'https://gitlab.gnome.org/GNOME/mobile-broadband-provider-info'
+  version '20201225'
```
Comment (uberhacker) [inline packages/mobile_broadband_provider_info.rb:6]: No `@_ver`?

File: packages/libnewt.rb [hunk 2]
```diff
@@ -0,0 +1,55 @@
+require 'package'
+
+class Libnewt < Package
+  description 'Not Eriks Windowing Toolkit - text mode windowing with slang'
+  homepage 'https://pagure.io/newt'
+  version '0.52.21'
```
Comment (uberhacker) [inline packages/libnewt.rb:6]: No `@_ver`?

File: packages/gpm.rb [hunk 3]
```diff
@@ -0,0 +1,82 @@
+require 'package'
+
+class Gpm < Package
+  description 'A mouse server for the console and xterm'
+  homepage 'https://www.nico.schottelius.org/software/gpm/'
+  version '1.20.7-e82d'
```
Comment (uberhacker) [inline packages/gpm.rb:6]: No `@_ver`?

Comment (satmandu) [reply packages/gpm.rb:6]: Not useful until the next tagged commit, right?

File: packages/mobile_broadband_provider_info.rb (hunk 1 above)
Comment (satmandu) [reply packages/mobile_broadband_provider_info.rb]: done

Comment (satmandu) [reply packages/mobile_broadband_provider_info.rb]: done

File: packages/gpm.rb (hunk 3 above)
Comment (uberhacker) [reply packages/gpm.rb:6]: Wouldn't this be the "correct" way to do it?
```
@_ver = '1.20.7'
version "#{@_ver}-e82d"
```

File: packages/gpm.rb [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+            [Service]
+            Type=forking
+            ExecStart=#{CREW_PREFIX}/bin/gpm -m /dev/input/mice -t imps2
+
+            [Install]
+            WantedBy=multi-user.target
+    GPM_SERVICE_EOF
+    IO.write("#{CREW_DEST_PREFIX}/systemd/system/gpm.service", @gpm_service)
+    FileUtils.chmod 0o644, "#{CREW_DEST_PREFIX}/systemd/system/gpm.service"
+    Dir.chdir CREW_DEST_LIB_PREFIX.to_s do
```
Comment (supechicken) [inline packages/gpm.rb:76]: `.to_s` not needed?

File: packages/gpm.rb [hunk 5]
```diff
@@ -0,0 +1,81 @@
...
+
+  def self.install
+    system "make DESTDIR=#{CREW_DEST_DIR} install"
+    FileUtils.mkdir_p "#{CREW_DEST_PREFIX}/systemd/system"
+    FileUtils.mkdir_p "#{CREW_DEST_PREFIX}/etc/profile.d"
+    FileUtils.mkdir_p "#{CREW_DEST_PREFIX}/etc/profile.d"
+    # via https://github.com/archlinux/svntogit-packages/tree/master/gpm/trunk
+    @gpm_sh = <<~GPM_SH_EOF
+      case \$( /usr/bin/tty ) in
+      /dev/tty[0-9]*) [ -n "\$(pidof -s gpm)" ] && disable-paste ;;
```
Comment (supechicken) [inline packages/gpm.rb:45]: Does this part necessary? I am thinking will this expression ever be matched in chrome os (the `tty*` is replaced by google's own virtual terminal call vt2, which use `/dev/pts/[0-9]` as their device file like crosh)

Comment (satmandu) [reply packages/gpm.rb:45]: Good call.

**Pull request merged**